benchmarking/write_output.py

```diff
@@ -0,0 +1,71 @@
+import requests
+import csv
+
+def get_next_response_entry(api_response_object_qps, api_to_proportion_of_requests_of_api):
+    """
+    Accepts one tuple of the form (API Name, Response Object, Current QPS).
+    Returns a Response entry to be filled in the ResponseEntry.csv file.
+
+    The entry includes:
+    1. APIName
+    2. ResponseTime
+    3. LoginQps
+    4. ViewUserQps
+    5. ListChatsQps
+    6. ListMessagesQps
+    7. CreateChatQps
+    8. CreateMessageQps
+    9. SignupQps
+    10. TotalQps
+    """
+
+    api_name = api_response_object_qps[0]
+    response_object = api_response_object_qps[1]
+    total_qps = api_response_object_qps[2]
+    
+    response_time = response_object.elapsed
+    login_qps = total_qps * api_to_proportion_of_requests_of_api["login"]
+    view_user_qps = total_qps * api_to_proportion_of_requests_of_api["viewUser"]
+    list_chats_qps = total_qps * api_to_proportion_of_requests_of_api["listChats"]
+    list_messages_qps = total_qps * api_to_proportion_of_requests_of_api["listMessages"]
+    create_chat_qps = total_qps * api_to_proportion_of_requests_of_api["createChat"]
+    create_message_qps = total_qps * api_to_proportion_of_requests_of_api["createMessage"]
+    signup_qps = total_qps * api_to_proportion_of_requests_of_api["signup"]
+
```

Comment:
This is too much repetitive code.
You can do this in a loop (including adding to the `response_entry` list.

Reply:
Done